examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -260,6 +270,67 @@ void ApplicationInit()
         sChipNamedPipeCommands.Stop();
     }
 
+    {
+        // operational state cluster
+        using OperationalStateStructType     = Clusters::OperationalState::Structs::OperationalStateStruct::Type;
+        using OperationalStateStructTypeList = DataModel::List<OperationalStateStructType>;
+        using PhaseListType                  = chip::CharSpan;
+        using PhaseList                      = chip::app::DataModel::List<const chip::CharSpan>;
+        using namespace Clusters::OperationalState;
+
+        const auto makeOperationalStateStructType = [](Clusters::OperationalState::OperationalStateEnum stateId, char * stateLabel,
+                                                       size_t len = 0) {
+            OperationalStateStructType state;
+            state.operationalStateID = stateId;
+            if (len)
+                state.operationalStateLabel = chip::CharSpan(stateLabel, len);
+            return state;
+        };
+
+        char opStopped[8]  = "Stopped";
+        char opRunning[32] = "Running";
+        char opPaused[64]  = "Paused";
+
+        OperationalStateStructType opS[3] = {
+            makeOperationalStateStructType(Clusters::OperationalState::OperationalStateEnum::kStopped, opStopped,
+                                           sizeof(opStopped)),
+            makeOperationalStateStructType(Clusters::OperationalState::OperationalStateEnum::kRunning, opRunning,
+                                           sizeof(opRunning)),
+            makeOperationalStateStructType(Clusters::OperationalState::OperationalStateEnum::kPaused, opPaused, sizeof(opPaused))
+        };
+        OperationalStateStructTypeList opL(opS);
+        Clusters::OperationalState::OperationalStateStruct op;
+        Clusters::OperationalState::OperationalErrorStateStruct opErr;
+
+        operationalstateServer.Init();
+        // set operational state list
+        operationalstateServer.SetOperationalStateList<OperationalStateStructType>(opL);
```

Comment:
So now `operationalstateServer` holds references to our stack, no?  How is this supposed to work?